### PR TITLE
[KOGITO-4323] remove leftover usages of workItem.getId()

### PR DESCRIPTION
**Pull request by mariofusco**

### CODE REVIEW

File: jbpm/jbpm-bpmn2/src/test/java/org/jbpm/bpmn2/concurrency/MultipleProcessesPerThreadTest.java [hunk 1]
```diff
@@ -144,33 +144,33 @@ public void start() {
```
Comment (danielezonca) [inline jbpm/jbpm-bpmn2/src/test/java/org/jbpm/bpmn2/concurrency/MultipleProcessesPerThreadTest.java:144]: This test is disabled with the message 
`@Disabled("This test costs time and resources, please only run locally for the time being.")`
Can you please run it locally to make sure the changes are safe?

File: jbpm/jbpm-flow/src/main/java/org/jbpm/process/instance/impl/demo/UIWorkItemHandler.java [hunk 2]
```diff
@@ -16,42 +16,31 @@
...
 import java.util.List;
 import java.util.Map;
 
-import javax.swing.JButton;
-import javax.swing.JFrame;
-import javax.swing.JList;
-import javax.swing.JPanel;
-import javax.swing.ListSelectionModel;
-import javax.swing.WindowConstants;
+import javax.swing.*;
```
Comment (danielezonca) [inline jbpm/jbpm-flow/src/main/java/org/jbpm/process/instance/impl/demo/UIWorkItemHandler.java:29]: Explicit imports

File: jbpm/jbpm-flow/src/main/java/org/jbpm/process/instance/impl/demo/UIWorkItemHandler.java [hunk 3]
```diff
@@ -16,42 +16,31 @@
 
 package org.jbpm.process.instance.impl.demo;
 
-import java.awt.BorderLayout;
-import java.awt.Dimension;
-import java.awt.GridBagConstraints;
-import java.awt.GridBagLayout;
-import java.awt.Insets;
+import java.awt.*;
```
Comment (danielezonca) [inline jbpm/jbpm-flow/src/main/java/org/jbpm/process/instance/impl/demo/UIWorkItemHandler.java:19]: Explicit imports

File: jbpm/jbpm-flow/src/main/java/org/jbpm/process/instance/impl/demo/UIWorkItemHandlerDialog.java [hunk 4]
```diff
@@ -16,25 +16,15 @@
 
 package org.jbpm.process.instance.impl.demo;
 
-import java.awt.BorderLayout;
-import java.awt.Dimension;
-import java.awt.GridBagConstraints;
-import java.awt.GridBagLayout;
-import java.awt.Insets;
+import java.awt.*;
```
Comment (danielezonca) [inline jbpm/jbpm-flow/src/main/java/org/jbpm/process/instance/impl/demo/UIWorkItemHandlerDialog.java:19]: Explicit imports

File: jbpm/jbpm-flow/src/main/java/org/jbpm/process/instance/impl/demo/UIWorkItemHandlerDialog.java [hunk 5]
```diff
@@ -16,25 +16,15 @@
...
 
-import javax.swing.JButton;
-import javax.swing.JDialog;
-import javax.swing.JLabel;
-import javax.swing.JPanel;
-import javax.swing.JTextArea;
-import javax.swing.JTextField;
-
-import org.kie.api.runtime.process.WorkItem;
+import javax.swing.*;
```
Comment (danielezonca) [inline jbpm/jbpm-flow/src/main/java/org/jbpm/process/instance/impl/demo/UIWorkItemHandlerDialog.java:26]: Explicit imports

File: jbpm/jbpm-bpmn2/src/test/java/org/jbpm/bpmn2/concurrency/MultipleProcessesPerThreadTest.java (hunk 1 above)
Comment (mariofusco) [reply jbpm/jbpm-bpmn2/src/test/java/org/jbpm/bpmn2/concurrency/MultipleProcessesPerThreadTest.java:144]: image

**Review by danielezonca — APPROVED** [review]

**Pull request merged**